src/gentropy/l2g.py

```diff
@@ -278,3 +279,45 @@ def _annotate_gold_standards_w_feature_matrix(self) -> L2GFeatureMatrix:
                 .persist()
             )
         raise ValueError("Dependencies for train mode not set.")
+
+
+class LocusToGeneEvidenceStep:
+    """Locus to gene evidence step."""
+
+    def __init__(
+        self,
+        session: Session,
+        locus_to_gene_predictions_path: str,
+        credible_set_path: str,
+        study_index_path: str,
+        evidence_output_path: str,
+        locus_to_gene_threshold: float = LocusToGeneEvidenceStepConfig().locus_to_gene_threshold,
```

Comment:
I think i know now why you are specifying the session to the config.I had this scenario before. And if you do `LocusToGeneEvidenceStepConfig()`, you are going to trigger Spark to get or create (in this case get) the session, which is prob not desirable. Because of that, I suggest not to get the default from the config

Reply:
Yes, exactly.

Reply:
So do you agree that the action here is not to pull the parameter from the config and remove the session reference?

Reply:
in case of the float you could not evaluate the `LocusToGeneEvidenceStepConfig`. Primitive types are defined at compile time if I am not mistaken.